src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/_help.py

```diff
@@ -276,3 +290,91 @@
     type: command
     short-summary: Place the CLI in a waiting state until a condition of the VM is met.
 """
+
+helps['disk'] = """
+    type: group
+    short-summary: Managed disk commands
+"""
+
+helps['snapshot'] = """
+    type: group
+    short-summary: Snapshot commands
+"""
+
+helps['image'] = """
+    type: group
+    short-summary: Manage custom virtual machine images based on managed disks/snapshots
+"""
+
+helps['disk create'] = """
+    type: command
+    short-summary: create a managed disk
+    examples:
+        - name: Create by importing from blob uri
+          text: az disk create -g myRG -n myDisk --source-blob-uri https://vhd1234.blob.core.windows.net/vhds/osdisk1234.vhd
+        - name: Create an empty disk
+          text: az disk create -g myRG -n myDisk --size-gb 10
+        - name: Create by copying from an existing disk or snapshot
+          text: az disk create -g myRG -n myDisk2 --source-disk myDisk
+"""
+helps['disk list'] = """
+    type: command
+    short-summary: list managed disks under a resource group or under current subscription
+"""
+helps['disk delete'] = """
+    type: command
+    short-summary: delete a managed disk
+"""
+helps['disk update'] = """
+    type: command
+    short-summary: update a managed disk including size and underlined storage account type
+"""
+helps['disk grant-access'] = """
+    type: command
+    short-summary: grant read access to a managed disk
+"""
+helps['disk revoke-access'] = """
+    type: command
+    short-summary: revoke read access to a managed disk
+"""
+
+helps['snapshot create'] = """
+    type: command
+    short-summary: create a snapshot
+    examples:
+        - name: Create by importing from blob uri
+          text: az snapshot create -g myRG -n mySnapshot --source-blob-uri https://vhd1234.blob.core.windows.net/vhds/osdisk1234.vhd
+        - name: Create an empty snapshot
+          text: az snapshot create -g myRG -n mySnapshot --size-gb 10
+        - name: Create by copying from an existing disk from the same resource group
+          text: az snapshot create -g myRG -n mySnapshot2 --source-disk myDisk
```

Comment:
same reason, a snapshot can be created from a disk or snapshot from the same group